src/features/game/lib/constants.ts

```diff
@@ -105,8 +104,8 @@ export const INITIAL_STOCK = (state?: GameState): Inventory => {
       "Kale Seed": new Decimal(36),
 
       "Apple Seed": new Decimal(12),
-      "Orange Seed": new Decimal(12),
-      "Blueberry Seed": new Decimal(12),
+      "Orange Seed": new Decimal(18),
+      "Blueberry Seed": new Decimal(24),
```

Comment:
We need to be careful doing large changes like this (x2). I do agree with the premise, but it also puts us in a tough position if we need to lessen the values (it feels like we are taking away something).

I would suggest the following to begin with:

Orange Seed: 16
Blueberry Seed: 20

This way we can ease into the change and assess the overall impact on BB usage + restocks

Reply:
@adamhannigan Thanks for your comments.  I have bump the numbers down for orange and blueberry seeds as suggested.